tutorials/getstarted.jl

```diff
@@ -0,0 +1,1220 @@
+
+This changes all functions that depend on the metric, most prominently the Riemannian matric, but also the exponential and logarithmic map and hence also geodesics.
+
+All functions that are not dependent on a metric – for example the manifold dimension, the tests of points and vectors we already looked at, but also all retractions – stay unchanged.
+This means that for example
+"""
+
+# ╔═╡ 93d77443-ee4c-40bd-bbc2-a669a269af9f
+[manifold_dimension(M₈₂), manifold_dimension(M₈)]
+
+# ╔═╡ 1cca1824-c983-40c0-a955-4ee4a32d7db1
+md"""
+both calls the same underlying function. On the other hand with
+"""
+
+# ╔═╡ 6c80af23-c960-4f5d-87e4-31535eb8a149
+p₅, X₅ = one(zeros(3, 3)), [1.0 0.0 1.0; 0.0 1.0 0.0; 1.0 0.0 1.0]
+
+# ╔═╡ efafedc3-4d25-4ce7-937c-71fd50d90143
+md"but for example the exponential map and the norm yield different results"
+
+# ╔═╡ cf13eee6-f53b-4109-afd6-8d2978a2aa07
+[exp(M₈, p₅, X₅), exp(M₈₂, p₅, X₅)]
+
+# ╔═╡ 3ca21132-6f53-4aba-9212-ce4f17d5b48c
+[norm(M₈, p₅, X₅), norm(M₈₂, p₅, X₅)]
+
+# ╔═╡ cfcf6cf4-6244-4b26-8097-90afb03abad6
+md"""
+Technically this done using Traits – the trait here is the [`IsMetricManifold`](https://juliamanifolds.github.io/Manifolds.jl/latest/manifolds/metric.html#Manifolds.IsMetricManifold) trait. Our trait system allows to combine traits but also to inherit properties in a hierarchical way, see [🔗 here](https://juliamanifolds.github.io/ManifoldsBase.jl/stable/decorator.html#Traits-with-an-inheritance-hierarchy) for the technical details.
+
+The same approach is used for
+
+* specifying a different [connection](https://juliamanifolds.github.io/Manifolds.jl/latest/manifolds/connection.html)
+* specifying a manifold as a certain [quotient manifold](https://juliamanifolds.github.io/Manifolds.jl/latest/manifolds/quotient.html)
+* specifying a certain [embedding](https://juliamanifolds.github.io/ManifoldsBase.jl/stable/decorator.html#The-Manifold-decorator)
+* specify a certain [group action](https://juliamanifolds.github.io/Manifolds.jl/latest/manifolds/group.html)
+
+Again, for all of these, the concrete types only have to be used if you want to do a second, different from the details, property, for example a second way to embed a manfiold. If a manifold is (in its usual representation) an embedded manifold, this works with the default manifold type already, since then it is again set as the reasonable default.
+"""
+
+# ╔═╡ d9a335f1-af0d-412e-bde2-cc147de90579
+md"""
+## References
+
+[^AbsilMahonySepulchre2008]:
+    > Absil, P.-A., Mahony, R. and Sepulchre R.,
+    > _Optimization Algorithms on Matrix Manifolds_
+    > Princeton University Press, 2008,
+    > doi: [10.1515/9781400830244](https://doi.org/10.1515/9781400830244)
+    > [open access](http://press.princeton.edu/chapters/absil/)
+[^AxenBaranBergmannRzecki2022]:
+	>Axen, S. D., Baran, M., Bergmann, R. and Rzecki, K:
+	> _Manifolds.jl: An Extensible Julia Framework for Data Analysis on Manifolds_,
```

Comment:
```suggestion
    >Axen, S. D., Baran, M., Bergmann, R. and Rzecki, K:
    > _Manifolds.jl: An Extensible Julia Framework for Data Analysis on Manifolds_,
```
Just replacing tabs with spaces.

Reply:
This might be also due to Pluto changing the format now and then (I excluded the Pluto notebooks fro the formatter since the default layout of Pluto yields a false in formatter and I do not want them to “fight”).

Reply:
OK, that's a very minor issue.